anax-in-container/Dockerfile_agbot.ubi

```diff
@@ -0,0 +1,50 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0
+
+ENV ANAX_LOG_LEVEL 3
+ENV DB_CONFIG '"DBPath": "/var/horizon"'
+ENV EXCHANGE_URL "http://localhost:8086/v1/"
+ENV EXCHANGE_ID "myorg/agbot"
+ENV EXCHANGE_TOKEN "abcdefg"
+ENV CSS_URL "http://localhost:8087/v1/"
+
+ARG DOCKER_VER=19.03.8
+
+# yum is not installed, use microdnf instead
+RUN microdnf update -y --nodocs && microdnf clean all
+
+# shadow-utils contains groupadd and adduser commands
+RUN microdnf install --nodocs -y shadow-utils \
+    && microdnf install --nodocs -y openssl ca-certificates \
+    && microdnf install -y wget iptables vim-minimal procps tar gettext \
+    && wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 \
+    && chmod +x ./jq \
+    && mv jq /usr/bin
+
+# install docker cli
```

Comment:
Do we need this in the agbot?

Reply:
I do not think we do. But old agbot container has docker cli installed.

Reply:
Removed.